### ISSUE TITLE
Change banner colour and title

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-accessibility-passport#85]
- Change banner colour to black or blue. 
- Replace all instances of "GC Accessibility Passport" with "GC Workplace Accessibility Passport"

**Comment by gobyrne** [comment]
I didn't know what solid color to use, but Rosalie suggested black or blue. @Yoni is that something you can choose for us? We also have at least 2 different banners, but I think both should be changed.
While it's not a "fail" for accessibility, it's recommended not to use the highest contract (white on black)

**Comment by brindasasi** [comment]
Please use banner as below 
https://www.canada.ca/en/government/publicservice/wellness-inclusion-diversity-public-service/diversity-inclusion-public-service/accessibility-public-service.html
